streams-contrib/streams-processor-urls/pom.xml

```diff
@@ -63,6 +63,7 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <!--
```

Comment:
What was the justification on commenting out these dependencies? If they're no longer needed, then we should remove them.